src/app/pages/utils/getIndexOfFirstNonBanner/index.ts

```diff
@@ -0,0 +1,8 @@
+import { Curation, VISUAL_STYLE } from '#app/models/types/curationData';
+
+export default (curations: Curation[]) => {
+  const checkIndex = curations.findIndex(
+    curation => curation.visualStyle !== VISUAL_STYLE.BANNER,
+  );
+  return checkIndex;
```

Comment:
We could technically make this a one-liner:
```suggestion
  return curations.findIndex(
    curation => curation.visualStyle !== VISUAL_STYLE.BANNER,
  );
```

Reply:
AND we could use destructuring to make it even simpler:

```suggestion
  return curations.findIndex(
    ({visualStyle})  => visualStyle !== VISUAL_STYLE.BANNER,
  );
```